src/components/wells/Sample.vue

```diff
@@ -89,12 +118,17 @@ export default {
 }
 
 .inactive {
-  background-color: gray;
   color: white;
+  background-color: gray;
 }
 
 .inspect {
   color: white;
   background-color: $well-red;
 }
+
```

Comment:
It might be worth considering a different colour as white on yellow sometimes does not provide sufficient contrast https://webaim.org/resources/contrastchecker/

Reply:
Thanks, done this.